typescript/nomad-deploy/scripts/staging/enroll.ts

```diff
@@ -0,0 +1,59 @@
+import { deploysToSDK } from '../../src/incremental/utils';
+import * as rinkeby from '../../config/testnets/rinkeby';
+import * as kovan from '../../config/testnets/kovan';
+import { deployNewChain } from '../../src/core';
+import { deployNewChainBridge } from '../../src/bridge';
+import { CoreDeploy, ExistingCoreDeploy } from '../../src/core/CoreDeploy';
+import {
+  BridgeDeploy,
+  ExistingBridgeDeploy,
+} from '../../src/bridge/BridgeDeploy';
+import { getPathToDeployConfig } from '../../src/verification/readDeployOutput';
+import { NomadContext } from '@nomad-xyz/sdk';
+import { enrollSpoke } from '../../src/incremental';
+
+const path = getPathToDeployConfig('staging');
+
+// Instantiate existing governor deploys on Rinkeby
+const rinkebyCoreDeploy = ExistingCoreDeploy.withPath(
+  rinkeby.chain,
+  rinkeby.stagingConfig,
+  path,
+);
+const rinkebyBridgeDeploy = new ExistingBridgeDeploy(
+  rinkeby.chain,
+  rinkeby.bridgeConfig,
+  path,
+);
+const rinkebyDomain = deploysToSDK(rinkebyCoreDeploy, rinkebyBridgeDeploy);
+
+// Deploy Kovan core and bridge with Rinkeby hub
+const kovanCoreDeploy = new CoreDeploy(kovan.chain, kovan.devConfig);
+const kovanBridgeDeploy = new BridgeDeploy(
+  kovan.chain,
+  kovan.bridgeConfig,
+  path,
+);
+deployNewChain(kovanCoreDeploy, rinkebyCoreDeploy);
```

Comment:
i think you'd need to await this 🤔  we should probably split these up into their own scripts -- core, bridge, and enroll

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/58/commits/98a1ba5ede55637af6718fc97fa1227ea13c323c